contrib/userscript/unnoticed.user.js

```diff
@@ -0,0 +1,207 @@
+// ==UserScript==
+// @name         [Unnoticed] Leaderboards
+// @namespace    https://github.com/christopher-dG/unnoticed
+// @version      0.2
+// @description  Display unranked leaderboard entries gathered by Unnoticed on their respective beatmap pages
+// @author       Node
+// @updateURL    https://github.com/christopher-dG/unnoticed/raw/master/contrib/userscript/unnoticed.user.js
+// @match        https://osu.ppy.sh/*/*
+// @include      https://p9bztcmks6.execute-api.us-east-1.amazonaws.com/*
+// @grant        GM_xmlhttpRequest
+// @require      https://code.jquery.com/jquery-3.2.1.min.js
+// @require      http://momentjs.com/downloads/moment.min.js
+// @connect      p9bztcmks6.execute-api.us-east-1.amazonaws.com
+// @run-at document-end
+// ==/UserScript==
+
+(function(){
+    'use strict';
+
+    var mods_enum = {
+        ''   : 0,
+        'NF' : 1,
+        'EZ' : 2,
+        ''   : 4,
```

Comment:
While not currently existing in the game, it's in the best interest of consistency to set this to 'NV'. Another fix could be to remove it entirely, as '4' should never be returned here in any form.

Reply:
NV is present on some really old scores, probably best to remove it.